go.mod

```diff
@@ -5,32 +5,32 @@ go 1.20
 require (
 	github.com/aws/aws-sdk-go v1.44.175
 	github.com/go-logr/logr v1.2.4
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.3.1
 	github.com/goombaio/namegenerator v0.0.0-20181006234301-989e774b106e
 	github.com/jetstack/cert-manager v1.7.1
-	github.com/kuadrant/authorino v0.10.0
-	github.com/kuadrant/kuadrant-operator v0.1.1-0.20230323151616-58593d01833a
+	github.com/kuadrant/authorino v0.15.0
+	github.com/laurafitzgerald/kuadrant-operator v0.0.0-20231103170836-625c57c4769f
```

Comment:
@laurafitzgerald fyi, you'd be better adding a replace line in here that way you don't have to update every import to test changes from your branch:

```
replace github.com/kuadrant/kuadrant-operator => github.com/laurafitzgerald/kuadrant-operator branhc-with-my-needed-chnages
```

Reply:
ah sweet. didn't know about this. so it wont error on the module name in this case?